packages/apps/auth0-database-scripts/src/get_user.ts

```diff
@@ -23,7 +23,9 @@ async function getUser(email: string): Promise<Auth0User | undefined> {
     const patronRecord = patronRecordResponse.result;
     return patronRecordToUser(patronRecord);
   } else if (patronRecordResponse.status === ResponseStatus.NotFound) {
-    return undefined;
+    // If a user is not found in sierra, we should be able to create the user in
+    // the next script that runs, 'create'
+    return;
```

Comment:
Will this not implicitly still be `undefined`? I'm willing to give it a shot though.

Reply:
I think you could be right, I think i've misread the get user function and thought it'll pass on the auth0 profile to create